### DISCUSSION TITLE
In-Network-Rates File: Schema and Definition of Plan Name and HIOS/EIN

### DISCUSSION THREAD

**Discussion by ghost** [CMSgov/price-transparency-guide#44] [Q&A] [answered]
In the in-network-rates schema, the schema lists

- Plan Name as a String defined as "The plan name and name of plan sponsor and/or insurance company."
- Plan ID as a String defined as "The 14-digit Health Insurance Oversight System (HIOS) identifier, or, if the 14-digit HIOS identifier is not available, the 5-digit HIOS identifier, or if no HIOS identifier is available, the Employer Identification Number (EIN)for each plan or coverage offered by a plan or issuer."

There are cases where a single Plan Name can point to multiple Plan ID values. For example there can be a Plan Name of "Just PPO Gold" that can point to multiple unique 14 digit HIOS values.

The current schema to support the 14 digit HIOS would require the Plan ID to be represented as an Array, or looking at the 14 digit HIOS values for the data the first 5 digits of the HIOS are the same. So in this case even though there are multiple unique 14 digit HIOS available and based on the current schema Type of String, should the 5 digit HIOS identifier be used since they are the same value and could be used in the Type of String for Plan ID?

For example using fake information,

**Plan Name,         Plan ID (14 digit),      Plan ID (first 5 digits)**
Just PPO Gold,    51236553678932,     51236
Just PPO Gold,    51236879676466,     51236
Just PPO Gold,    51236987667232,     51236

using the existing schema we would use 51236 in the field for Plan ID?

**Comment by tdfow** [comment]
It is my understanding that if the 14 digit HIOS ID is available and unique then it would be used in reporting, resulting in 3 files in your example.   If there is a unique HIOS ID, then wouldn't that indicate some difference, so wouldn't you have to refer to the Plan Name as Just PPO Gold 1 or Just PPO Gold 2 or Just PPO Gold 3--if they had all unique HIOS IDs?   In your example, would all the providers and rates being published be the same for the Plan Name Just PPO Gold--regardless of HIOS ID?

**Comment by ghost** [comment]
The providers and rates are the same for the plan name of Just PPO Gold (the provider network type being PPO). The only differentiators causing the various HIOS values is because the premium rates may differ, the benefits or cost share or variables (dep 29, Contraceptives, etc) make it unique.

So either 

- the Plan ID goes from String to Array 
- use the first 5 for a String
- need as you pointed out concatenate a value, like 01, onto the plan name used in the file naming and the Plan Name schema field knowing any Plan Name/HIOS ID submissions to the state or federal agencies would not be exact if they were comparing Plan Names for the HIOS ID

**Reply by tdfow** [reply]
Based on the mandate wording--- I would think it would still need to be the 14 digit HIOS ID--if a 14 digit exists, so then either the Plan ID would be an array or need to have a unique Plan Name.  I have seen this type of example too.

**Reply by angelamarie0975** [reply]
We have the same question regarding plans - can we update the file schema to make this an array for the PLAN_ID attribute if this is allowed? In our case, we have a base product 'Health Option' and this product would have many different plans that utilize the same network of providers - 'Health Option 1', 'Health Option 2', 'Health Option 3'...). Could we use the Product Name of Health Option for the Plan Name and have the Plan ID be an array of those associated HIOS? This is going to make a difference between say 1000 individual files (based on the naming convention per plan) vs say only 30 files for the different products we offer. We need to know if we are allowed to make descisions on the file layout based on some of these questions that we have. I have a similar question around place of service code.

**Reply by ghost** [reply]
I agree, we are processing some of our data that is similar to your example and then "fanning" out a "base" plan into multiple plan files where we would just change the plan name and associated plan id (HIOS).

**Reply by ghost** [reply]
@shaselton Could you provide feedback on how this could be implemented in the examples I provided based on a change in the schema or file naming convention. The payor I am working with does have this situation and we need to provide a design for it to be implemented. 

We are leaning towards the the Plan ID goes from String to Array which would be more efficient and flow the plan naming convention. 

Are we tied to the schema that is listed here on Github? Or can we create a variant as it would meet the requested data in the rule?

**Reply by ghost** [reply]
@angelamarie0975 Have you received a CMS response? What are you planning on doing?

**Reply by angelamarie0975** [reply]
@BobSyracuse No, I have not heard back from them yet. 

I am not fully clear on who is supposed to be the contact from CMS to provide this guidance? I just listed all of the names in the contributor list on the main page below. 

@shaselton 
@shaselton-usds
@rogelyn-cms
@tdfow 
@StoneCypher

**Reply by tdfow** [reply]
I think @shaselton , @shaselton-usds is associated with CMS, as they have responded/closed some of my topics.   
I am not. :)

**Reply by bryanbriegel** [reply]
@BobSyracuse  - if helpful, here is a breakdown of 14 byte HIOS plan IDs.  The first 5 are the HIOS Issuer-ID.  The Product ID would be bytes 1-10.  That may be the level of interest if the 14 byte plan is not. 

14-byte HIOS plan ID:  This is the plan ID.
Example:
12345VA1239807
HIOS Issuer ID (or Issuer ID): The first 5 bytes. 
12345
State: where the plan is offered: 2-byte state code is appended to the HIOS issuer ID: 
12345VA
Product ID:  a 3-byte code is appended to the Issuer ID/State code to define the Product ID.
12345VA123
Plan ID:  a 4-byte code is appended to the Product ID to define the Plan ID.
12345VA1239807

**Reply by ghost** [reply]
@bryanbriegel Thank you for posting the information! Just was looking for the CMS guidance on how to handle multiple unique HIOS that point to the same payor plan name and how this would be handled based on the existing Github file name convention and In-Network schema.

**Reply by shaselton-usds** [reply]
@BobSyracuse -- sorry about the delay on this one, there is actually a reason for it. What I can say here/now is that we can expect rule guidance on this front (HIOS/EIN) in the near future. Once it is out, I'll point the community to it.

@bryanbriegel is spot on with the HIOS documentation. Thanks for posting that. Here is documentation from CMS on how the ID is broken down if interested, first paragraph: https://www.cms.gov/CCIIO/Resources/Forms-Reports-and-Other-Resources/Downloads/HIOS-Plan-Finder-User-Manual.pdf#page=57

**Reply by angelamarie0975** [reply]
@shaselton 
Do we have any updates on this subject? This is holding up some of our implementation because we are needing answers for this still.

**Reply by mmorri05** [reply]
Bumping for any new information.
Has the question been resolved regarding how the physical file/s for one plan for any one of the three types be generated?
Is the direction to combine/concatenate all data for a single "reporting_entity_name" into a single physical file as I see it when I read the FR?
Or...  is it allowed to split the data into multiple files based on the "plan_id"?
If the latter, where in the FR does it allow that?

**Reply by MMath19103** [reply]
> @BobSyracuse -- sorry about the delay on this one, there is actually a reason for it. What I can say here/now is that we can expect rule guidance on this front (HIOS/EIN) in the near future. Once it is out, I'll point the community to it.
> 
> @bryanbriegel is spot on with the HIOS documentation. Thanks for posting that. Here is documentation from CMS on how the ID is broken down if interested, first paragraph: https://www.cms.gov/CCIIO/Resources/Forms-Reports-and-Other-Resources/Downloads/HIOS-Plan-Finder-User-Manual.pdf#page=57

Hi - any updates on the updated rules?

**Reply by ghost** [reply]
@MMath19103 I was told by an individual in my organization that a CMS update should be released this week.

**Reply by MMath19103** [reply]
> @MMath19103 I was told by an individual in my organization that a CMS update should be released this week.

Thanks Bob - are we looking for an update just for the EIN/HIOS or for plan / plan id ?  I've lost track :-)

**Reply by ghost** [reply]
@MMath19103 I would have liked for all the above :-) But having not heard or seen anything this week, I question the validity of the source. I think we are all going to be stuck with what information we have for the 1/1/22 due date.

**Comment by RLTx1391** [comment, marked as answer]
> Has anyone solutioned for cases where a single Plan Name can point to multiple unique 14 digit HIOS values?
@shaselton-usds you mentioned we expect rule guidance on this front in the near future. Is that still the case?

**Reply by angelamarie0975** [reply]
We have this scenario. Since the plan_id is where the HIOS information is and it is a part of the reporting plans object - a unique record is actually -  plan_name + plan_id_type + plan_id + plan_market_type combined, so by having a different HIOS in that combination, it would make for a unique record in that array of plans. They will just happen to all have the same plan name, but their different HIOS or EIN if applicable, making it unique. This was one of the reasons for the TOC file to report these types of plans. Not sure if that helps at all?

**Reply by shaselton-usds** [reply]
@angelamarie0975 @RLTx1391 @BobSyracuse 

There is a change in this PR: https://github.com/CMSgov/price-transparency-guide/pull/447 that allows reporting at the 10-digit level vs the 14-digit level. 

@angelamarie0975 is correct that plan_name + plan_id_type + plan_id + plan_market_type is the composite key for uniqueness in plan reporting -- hopefully the new 10-digit level and using the table of contents can point to a single file to cut down on the duplicate reporting.

**Reply by amyxguo** [reply]
how come plan_id alone won't be enough to uniquely identify plan reporting?

**Reply by JR-CMS** [reply]
The plan_id (when reflected as an EIN) is not sufficient to reliably identify the plan sponsor (i.e., employer), because there is no public way to look up an EIN to determine the common name of the entity it represents. In addition, the information disclosed in plan_name attribute is often that of the insurance company and not the plan sponsor. CMS is aware of this issue and is considering solutions to better identify plans in the MRFs.
